modules/creating-logfilesmetricexporter.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/cluster-logging-collector.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-logfilesmetricexporter_{context}"]
+= Creating a LogFileMetricExporter resource
+
+In {logging} version 5.8 and newer versions, the `LogFileMetricExporter` custom resource (CR) is no longer deployed with the collector by default.
+You must manually create a `LogFileMetricExporter` CR to generate metrics from the logs produced by running containers.
+
+If you do not create the `LogFileMetricExporter` CR, you may see a *No datapoints found* message in the {product-title} web console dashboard for *Produced Logs*.
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have installed the {clo}.
+* You have installed the {oc-first}.
+
+.Procedure
+
+. Create a `LogFileMetricExporter` CR as a YAML file:
++
+.Example `LogFileMetricExporter` CR
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1alpha1
+kind: LogFileMetricExporter
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  nodeSelector: # <1>
```

Comment:
it is better to provide an example using same nodeSelector/tolerations  as the collection section in clusterlogging/instance and add a note.
For example:
1)  default  LogFileMetricExporter without nodeSelector and /tolerations
2)  deploy LogFileMetricExporter  to infra workers.

Reply:
Reverting to what was provided in https://github.com/openshift/openshift-docs/pull/70680#discussion_r1464228958

If there's anything else specific required, please add it as a suggestion to the PR and provide sample code / notes.